SALib/sample/latin.py

```diff
@@ -39,8 +39,23 @@ def sample(problem, N):
         for j in range(N):
             result[j, i] = temp[j]
 
-    scale_samples(result, problem['bounds'])

```

Comment:
Could all this functionality be pushed into the `scale_samples` function if the problem dict was passed to it?  This way, there would be no duplication of the code across the different methods?